### DISCUSSION TITLE
Support copy operation on alert rule list item

### DISCUSSION THREAD

**Discussion by yizishou** [grafana/grafana#45579] [Alerting]
Thanks. This will be a very helpful feature to us, when creating similar complex alert rule.
![image](https://user-images.githubusercontent.com/20429728/154643374-cc765817-9f66-4822-a503-8f2175efca8a.png)

Either creating a new rule item background with a new name, or opening a creating page with data fullfilled, all ok.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
